### PR TITLE
filter guest component

### PR DESCRIPTION
**Pull request by Zerowhizkey**
yo fixed something look at it

### CODE REVIEW

File: src/types/global.d.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ interface restrictionsProp {
     setVegan: React.Dispatch<React.SetStateAction<boolean>>;
     setVegetarian: React.Dispatch<React.SetStateAction<boolean>>;
 }
+interface guestProp {
```
Comment (linda-94-andersson) [inline src/types/global.d.ts:12]: Choice one! For this guestProp or guestList, only one is needed.

File: src/components/GuestFilter/GuestFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { CardContainer, CardTitle } from './GuestFilter.styles';
+
+const GuestFilter = ({ guest }: guestProp) => {
```
Comment (linda-94-andersson) [inline src/components/GuestFilter/GuestFilter.tsx:3]: If you change prop, remember t change it here too.

**Review by linda-94-andersson — CHANGES_REQUESTED** [review]
Well done! Don't forget to check changes.

File: src/types/global.d.ts (hunk 1 above)
Comment (Zerowhizkey) [reply src/types/global.d.ts]: i fix  i fix

**Review by linda-94-andersson — APPROVED** [review]
Very good!

**Pull request merged**